cit3.0-web/src/components/Page/Datasources/Datasources.js

```diff
@@ -0,0 +1,248 @@
+  },
+  {
+    "Data Source": "Communities",
+    "Metadata URL":
+      "COMMUNITIES_V6.csv - should this be: https://open.canada.ca/data/en/dataset/fe945388-1dd9-4a4a-9a1e-5c552579a28c",
+  },
+  {
+    "Data Source": "Court Locations",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/c95a2ad5-f62a-43d6-8678-80a617b6200e",
+  },
+  {
+    "Data Source": "Current Census Regions",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/1aebc451-a41c-496f-8b18-6f414cde93b7",
+  },
+  {
+    "Data Source": "Customs Port of Entry",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/4fac3ad6-8749-4741-ac98-527b23e4b0b2",
+  },
+  {
+    "Data Source": "Diagnostic Facilities",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/4f75e9f6-1459-4ae2-990d-b53b1c389525",
+  },
+  {
+    "Data Source": "Digital Road Atlas (DRA)",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/bb060417-b6e6-4548-b837-f9060d94743e",
+  },
+  {
+    "Data Source": "Emergency Social Services Facilities",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/8473689f-2d06-4e84-ac8b-e91033fd0f0b",
+  },
+  {
+    "Data Source": "Environmental Assessment Office (EAO) Points",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/bc1c5380-6fca-4666-996b-049ad8ca04cc",
+  },
+  {
+    "Data Source": "First Nations Reserves",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/c2ce81af-78c1-467c-b47e-c392cd0a771f",
+  },
+  {
+    "Data Source": "First Responders",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/652c49eb-7295-4ae2-8f26-39103f23b50d",
+  },
+  {
+    "Data Source": "Freshwater Atlas Lakes",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/cb1e3aba-d3fe-4de1-a2d4-b8b6650fb1f6",
+  },
+  {
+    "Data Source": "Freshwater Atlas Rivers",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/f7dac054-efbf-402f-ab62-6fc4b32a619e",
+  },
+  {
+    "Data Source": "Health Authority Boundaries",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/7bc6018f-bb4f-4e5d-845e-c529e3d1ac3b",
+  },
+  {
+    "Data Source": "Hexagonal Grid",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/a3badeec-feed-41d9-b33d-5ae41c19f27d",
+  },
+  {
+    "Data Source": "Hospitals",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/383eaf98-afd7-436a-9556-67ecf14f64a7",
+  },
+  {
+    "Data Source": "Laboratory Services in BC",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/401671c5-4fc4-4ffb-9e29-0cdc3c9433ba",
+  },
+  {
+    "Data Source": "Local Government Offices",
+    "Metadata URL":
+      "https://catalogue.data.gov.bc.ca/dataset/0fe8083a-881a-4fa8-8925-3a23b788e1b9",
+  },
+  {
+    "Data Source": "Major Processing Timber Processing Facilities",
```

Comment:
Is that really what it's called?  That's a lot of processing...

Reply:
No idea, I will check with Dan H.